src/main/scala/me/shadaj/romeojuliet/Tragedy.scala

```diff
@@ -21,6 +21,53 @@ case class YouHaveToMarry(that: ActorRef)
 case object HelpMe
 
 case object SleepingPotion
+case class WakingUp(actorRef: ActorRef)
+
+object Tragedy {
+  final val Name = "tragedy"
+  def props: Props = Props(new Tragedy)
+}
+
+/* The single top-level actor. */
+class Tragedy extends Actor {
+
+  override val supervisorStrategy = SupervisorStrategy.stoppingStrategy
+
+  val romeo = context.watch(context.actorOf(Romeo.props, Romeo.Name))
```

Comment:
We have to watch all child actors so we can terminate the system if one (Romeo) dies. Else the app won't stop.